FortnitePorting.Plugins/Blender/fortnite_porting/processing/import_context.py

```diff
@@ -967,16 +967,26 @@ def setup_params(mappings, target_node, add_unused_params=False):
                     diffuse_node = shader_node.inputs["Diffuse"].links[0].from_node
                     links.new(diffuse_node.outputs[0], shader_node.inputs["Emission Multiplier"])
 
+                if get_param(switches, "Use Engine Colorized GMap"):
+                    gmap_node = nodes.new(type="ShaderNodeGroup")
+                    gmap_node.node_tree = bpy.data.node_groups.get(".FP GMap Material")
+                    gmap_node.location = -1100, -69
+                    if len(shader_node.inputs["Diffuse"].links) > 0:
+                        nodes.remove(shader_node.inputs["Diffuse"].links[0].from_node)
+                    links.new(gmap_node.outputs[0], shader_node.inputs[0])
+                    setup_params(gmap_material_mappings, gmap_node)
+
                 if get_param(switches, "useGmapGradientLayers"):
                     gradient_node = nodes.new(type="ShaderNodeGroup")
                     gradient_node.node_tree = bpy.data.node_groups.get("FP Gradient")
                     gradient_node.location = -500, 0
                     nodes.remove(shader_node.inputs["Diffuse"].links[0].from_node)
                     links.new(gradient_node.outputs[0], shader_node.inputs[0])
 
-                    gmap_node = nodes.new("ShaderNodeValue")
-                    gmap_node.location = -1000, -120
-                    gmap_node.outputs[0].default_value = 0.5
+                    gmap_node = nodes.new(type="ShaderNodeGroup")
+                    gmap_node.node_tree = bpy.data.node_groups.get(".FP GMap")
+                    gmap_node.location = -1120, -240
+                    setup_params(gmap_mappings, gmap_node, False)
```

Comment:
Changed this from using a static 0.5 value to the G channel of the M texture to better match the UE material and add more detail

![Before](https://github.com/user-attachments/assets/71a8434d-47e4-40fa-9f3a-992a20fe87a2)
![After](https://github.com/user-attachments/assets/61395fd2-e374-4722-be6c-7710b2f34c7f)